rke/k8s_rke_system_images.go

```diff
@@ -5012,11 +5092,10 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.6"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.13"),
 		},
-		// Enabled out of band post v2.5.7
-		// Reminder: This template contains ACI images which aren't in templates for v2.4.x
-		"v1.19.9-rancher1-2": {
+		// Enabled in v2.5.8
+		"v1.19.10-rancher1-1": {
```

Comment:
Removed the `// Reminder: This template contains ACI images which aren't in templates for v2.4.x` comment from v1.19.10-rancher1-1 and v1.20.6-rancher1-1 because we don't have any v1.19.x or v1.20.x templates in v2.4.x so it's not relevant.